src/architecture/_GeneralModuleFiles/sys_model.i

```diff
@@ -11,27 +11,32 @@ from Basilisk.architecture.swig_common_model import *
 %include "swig_conly_data.i"
 %include "architecture/utilities/bskLogging.h"
 
-%feature("director") SysModel;
-%feature("pythonappend") SysModel::SysModel %{
-    self.__super_init_called__ = True%}
-%rename("_SysModel") SysModel;
 %include "sys_model.h"
 
-%pythoncode %{
-class SuperInitChecker(type):
-
-    def __call__(cls, *a, **kw):
-        rv = super(SuperInitChecker, cls).__call__(*a, **kw)
-        if not getattr(rv, "__super_init_called__", False):
-            error_msg = (
-               "Need to call parent __init__ in SysModel subclasses:\n"
-               f"class {cls.__name__}(sysModel.SysModel):\n"
-               "    def __init__(...):\n"
-               "        super().__init__()"
-            )
-            raise SyntaxError(error_msg)
-        return rv
-
-class SysModel(_SysModel, metaclass=SuperInitChecker):
-    bskLogger: BSKLogger = None
+%pythonbegin %{
+from typing import Union, Iterable
+from Basilisk.utilities import pythonVariableLogger
 %}
+
+%extend SysModel
+{
+    %pythoncode %{
+        def logger(self, variableNames: Union[str, Iterable[str]], recordingTime: int = 0):
```

Comment:
Do we want to default this to 0? It won't ever record. I'm wondering if 1 is better, as it'll at least record something. I'm curious to know how this type of behavior is done elsewhere in Basilisk, though. 

I think this also applies to the previous commit.